extensions/blocks/podcast-player/edit.js

```diff
@@ -259,6 +261,12 @@ const PodcastPlayerEdit = ( {
 		);
 	}
 
+	const updateColor = ( colorName, handler ) => ( color ) => {
```

Comment:
Two suggestions here:
1. Can we use e.g. `createColorChangeHandler` name for this func to describe its purpose better?
2. Can we derive the `handler` too here as we do with the `attr` value?, e.g.:

   ```js
	const createColorSetter = colorName => color => {
		const name = startCase( colorName );
		const attr = `hex${ name }Color`;
		const handler = props[ `set${ name }Color` ];
		/*
		...or if we want to keep all the props destructured:
		const handler = {
			setPrimaryColor,
			setSecondaryColor,
			setBackgroundColor,
		}[ `set${ name }Color` ];
		*/

		setAttributes( { [ attr ]: color } );
		handler( color );
	};
   ```

Reply:
> 1. Can we use e.g. `createColorChangeHandler` name for this func to describe its purpose better?

👍 

> 2. Can we derive the handler too here as we do with the attr value?, e.g.:

I thought the same but I considered it as too much :mmagic: Taking a look now.

Reply:
Do we need to get all the block properties in the constant 'props' for this? And then we need to destructuring them? No sure, tbh.
And although the second option works, I think it becomes more unreadable compared to the current approach.
What do you think

Reply:
> And although the second option works, I think it becomes more unreadable compared to the current approach.

It's a minor, but for me, it's an issue of consistency - if we decide to derive stuff as we do with the `attr` name it would be nice to keep the pattern for deriving other values too - that's why I suggested doing it for the handler func as it uses the same naming pattern. Otherwise, we're importing the `startCase` here just for one call which we don't really need, because we could do as well:

```js
 const createColorChangeHandler = (attr, handler) => color => {
 	setAttributes( { [ attr ]: color } );
 	handler( color );
 };
```

...and pass the attr value explicitly (which would make things even more readable, I guess 😄): 

```js
onChange={ createColorChangeHandler( 'hexPrimaryColor', setPrimaryColor ) }
```

What do you think?

Reply:
it's ok for me :-D

Reply:
@WunderBart https://github.com/Automattic/jetpack/pull/15335 :-)